package.js

```diff
@@ -19,7 +19,7 @@ Package.onTest(function(api) {
 function configure(api) {
   api.versionsFrom('1.0');
   api.use('underscore');
-  api.use('react@0.1.4');
+  api.use('react@0.1.4', {weak: true});
```

Comment:
I like to have something more from this.
We need to depend on the other package as well.

Then, we can detect the React namespace from the these packages.
I mean something like this: https://github.com/kadirahq/meteor-react-layout/blob/master/lib/react_layout.js#L28